x-pack/plugins/secops/public/components/direction/index.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as React from 'react';
+import { pure } from 'recompose';
+
+import { NetworkDirectionEcs } from '../../graphql/types';
+import { DraggableBadge } from '../draggables';
+import { NETWORK_DIRECTION_FIELD_NAME } from '../source_destination/network';
+
+export const INBOUND = 'inbound';
+export const OUTBOUND = 'outbound';
+
+export const EXTERNAL = 'external';
+export const INTERNAL = 'internal';
+
+export const INCOMING = 'incoming';
+export const OUTGOING = 'outgoing';
+
+export const LISTENING = 'listening';
+export const UNKNOWN = 'unknown';
+
+export const DEFAULT_ICON = 'questionInCircle';
```

Comment:
Not with this PR, but you can eventually use some of the types from `IconType`, from Eui for some of these to help you out in case they change them and you can get a compiler error.

Example where this would work:

```ts
import { IconType } from '@elastic/eui';

export const DEFAULT_ICON: IconType = 'questionInCircle';
```

Reply:
Although we do get a compiler error with the code as-is if an invalid icon type is used, I agree the narrower types are preferable. I'd like to do a separate PR that replaces all instances of the existing pattern above with the better approach you suggested here.